### ISSUE TITLE
[BUG] rapids-tools v22.04.0 release jar reports maven dependency issue :  rapids-4-spark-common_2.12:jar:22.04.0 NOT FOUND

### ISSUE THREAD

**Issue by NvTimLiu** [NVIDIA/spark-rapids#5233]
rapids-tools v22.04.0 release jar reports maven dependency issue :  rapids-4-spark-common_2.12:jar:22.04.0 `NOT FOUND`

CLI: 
```
mvn dependency:get -B -DremoteRepositories=https://oss.sonatype.org/content/repositories/staging -DgroupId=com.nvidia -DartifactId=rapids-4-spark-tools_2.12 -Dversion=22.04.0
```

Error log:
```
09:58:55  [ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:2.8:get (default-cli) on project standalone-pom: Couldn't download artifact: Missing:
09:58:55  [ERROR] ----------
09:58:55  [ERROR] 1) com.nvidia:rapids-4-spark-common_2.12:jar:22.04.0
09:58:55  [ERROR] 
09:58:55  [ERROR] Try downloading the file manually from the project website.
09:58:55  [ERROR] 
09:58:55  [ERROR] Then, install it using the command:
09:58:55  [ERROR] mvn install:install-file -DgroupId=com.nvidia -DartifactId=rapids-4-spark-common_2.12 -Dversion=22.04.0 -Dpackaging=jar -Dfile=/path/to/file
09:58:55  [ERROR] 
09:58:55  [ERROR] Alternatively, if you host your own repository you can deploy the file there:
09:58:55  [ERROR] mvn deploy:deploy-file -DgroupId=com.nvidia -DartifactId=rapids-4-spark-common_2.12 -Dversion=22.04.0 -Dpackaging=jar -Dfile=/path/to/file -Durl=[url] -DrepositoryId=[id]
09:58:55  [ERROR] 
09:58:55  [ERROR] Path to dependency:
09:58:55  [ERROR] 1) org.apache.maven.plugins:maven-downloader-plugin:jar:1.0
09:58:55  [ERROR] 2) com.nvidia:rapids-4-spark-tools_2.12:jar:22.04.0
09:58:55  [ERROR] 3) com.nvidia:rapids-4-spark-common_2.12:jar:22.04.0
09:58:55  [ERROR] 
09:58:55  [ERROR] ----------
09:58:55  [ERROR] 1 required artifact is missing.
```

Root cause: 
rapids-tools:v22.04.0 release candidates depends on `rapids-4-spark-common_2.12:jar:22.04.0`, but we won't release apids-4-spark-common_2.12:v22.04.0 to maven central: 

https://github.com/NVIDIA/spark-rapids/blob/branch-22.04/tools/pom.xml#L45 
```
        <dependency>
            <groupId>com.nvidia</groupId>
            <artifactId>rapids-4-spark-common_${scala.binary.version}</artifactId>
            <version>${project.version}</version>
        </dependency>
```

**Comment by tgravescs** [comment]
this was broken by https://github.com/NVIDIA/spark-rapids/pull/4408 where it added the dependency.  we could actually leave that but we would need to generate the dependency reduced pom file and push that instead.
I believe the fix proposed in the PR is to just copy the one class we need, which is also fine and makes sure we don't pick up other things in the common jar we don't need.

**Comment by tgravescs** [comment]
fix merged to 22.04
